tests/io/FileFactoryTest.py

```diff
@@ -0,0 +1,47 @@
+import os
+import unittest
+
+from coalib.io.FileFactory import FileFactory
+
+
+class FileFactoryTest(unittest.TestCase):
+
+    def setUp(self):
+        file_factory_test_dir = os.path.join(os.path.split(__file__)[0],
+                                             'FileFactoryTestFiles')
+
+        self.test_file = os.path.join(file_factory_test_dir, 'test.txt')
+        self.file_factory_object = FileFactory(self.test_file)
```

Comment:
Optional: To keep things shorter, people refer to the thing to test as `uut` = unit under test. But I also have nothing against longer more descriptive names :)